### PR TITLE
Refactored troubleshooting and added SMTP port resolution

### LINKED ISSUE #8: Suggestion for 'Email confirmation isn't working' section
In completing the setup work I encountered an issue where email wasn't being sent - the host I'm using, Scaleway, blocks the *standard* SMTP ports. For Mailgun (and Sparkpost supports this) I used port 2525 instead and everything worked well.

**TroubleShooting Section Suggestion**
1. Add changing SMTP port to 2525 should email send failure occur.
1. Add mention of Docker commands to see container logs, such as: ``docker logs mastodon_web_1``

Happy to create a pull request if you'd prefer.

### PR DESCRIPTION
**Pull request by jrimmer**
Fixes #8

### CODE REVIEW

**Comment by ummjackson** [comment]
Looks like this change is captured in https://github.com/ummjackson/mastodon-guide/pull/11 too? Is this PR required?

**Comment by jrimmer** [comment]
I'll correct the other to only have the related commit - to keep things straight.

**Comment by ummjackson** [comment]
Perfect, thanks.

Cheers,
*Jackson.*


On Sun, Apr 23, 2017, at 08:25 PM, Jason Rimmer wrote:
> I'll correct the other to only have the related commit - to keep
> things straight.> — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub[1], or mute the thread[2].> 

Links:

  1. https://github.com/ummjackson/mastodon-guide/pull/12#issuecomment-296516307

File: up-and-running.md [hunk 1]
```diff
@@ -302,8 +302,21 @@ sudo systemctl restart nginx.service
...
 
 After your instance is live for ~24 hours, you shouldn't be having this issue any longer. 
+
+#### Is your SMTP sending port blocked?
+
+Some ISPs and hosts such as Scaleway block email sending on the standard SMTP port, `25`. To see if that's happening view the web container's logs using `docker logs mastodon_web_1` and look for connection timeouts. To correct this edit the `.env.production` configuration file and change `SMTP_PORT: 25` to port `2525`, which Mailgun and SparkPost support.
+
+After doing so you'll need to rebuild the containers:
+```
+docker-compose down
```
Comment (ummjackson) [inline up-and-running.md]: Please update this to `stop` instead of `down` as per comment on other PR

**Comment by jrimmer** [comment]
Ready for review/merge.

**Comment by ummjackson** [comment]
Perfect! Merging.

**Pull request merged**